docs/sources/mimir/get-started/_index.md

```diff
@@ -12,7 +12,7 @@ weight: 10
 You can get started with Grafana Mimir _imperatively_ or _declaratively_:
 
 - **Imperatively**: The written instructions that follow contain commands to help you start a single Mimir process. You would need to perform the commands again to start another Mimir process.
-- **Declaratively**: The following video tutorial uses `docker-compose` to deploy multiple Mimir processes. Therefore, if you want to deploy multiple Mimir processes later, the majority of the configuration work will have already been done.
+- **Declaratively**: The following video tutorial uses `docker-compose` to deploy multiple Mimir processes. Therefore, if you want to deploy multiple Mimir processes later, the majority of the configuration work will have already been done. However, the tutorial directory path has been changed to `docs/sources/mimir/get-started/play-with-grafana-mimir` after the release of the video below.
```

Comment:
Perhaps we make this a note so that it's more prominent?

I've removed the use of "below" because that directional language doesn't always work when the page is read by a screen reader or similar accessibility tools.

```suggestion
- **Declaratively**: The following video tutorial uses `docker-compose` to deploy multiple Mimir processes.
  Therefore, if you want to deploy multiple Mimir processes later, the majority of the configuration work will have already been done.
  
  {{< admonition type="note" >}}
  The tutorial directory path changed to `docs/sources/mimir/get-started/play-with-grafana-mimir` after the release of the video.
  {{< /admonition >}}
```